lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,200 @@
+
+**For Example:** School A and B both have basketball teams. The 6 foot kids on School A's team are better than the 6 foot kids on School B's team. The 5' 10" kids from School A are also better than 5' 10" kids from School B. In fact, at every height, the kids from School A are better than the kids from School B. __But School B's team is better than School A.__ How is this possible? School B has __more tall kids__ than School A!
+
+[.lesson-instruction]
+- We need to break this dataset up into _grouped samples_, so that all of the data for Rhode Island is in one table, all of the data for Maine is in another, etc.
+- Working in pairs or small groups, complete the rest of of @printable-exercise{exploring-covid.adoc}.
+
+How does the `filter` function work?
+
+__The `filter` function consumes a Table and **a helper function!** The helper function is used on every Row of the Table, producing true or false. The `filter` function takes all the Rows for which the helper produced true, and combines them all into a new table.__
+
+[.strategy-box, cols="1", grid="none", stripes="none"]
+|===
+
+|
+@span{.title}{More Practice with Filtering}
+
+While filtering is introduced in this lesson, the primary goal is about exponential functions. If your students need more practice with filtering - or wish to filter their own datasets - we recommend checking out the @lesson-link{table-functions} lesson.
+|===
+
+We can use `filter` to split our dataset into multiple sub-groups, and then fit models to each sample.
+
+[.lesson-instruction]
+Create helper functions to filter at least three other states, then use them to define the Covid tables for each state.
+
+=== Common Misconceptions
+
+It's extremely common for students to think that filtering a table *changes the original table*. This is NOT how it works in Pyret! Instead, the `filter` function always produces a _new_ table, containing only the Rows for which the supplied function evaluates to `true`
+
+=== Synthesize
+- In what other situations would it be useful to filter a dataset?
+- Can you think of other examples where Simpson's Paradox might arise?
+
+
+== Looking for Patterns @duration{15 minutes}
+
+=== Overview
+
+Students explore their newly-filtered `MA-covid` dataset, trying to fit different kinds of models to it. This section makes heavy use of @link{https://www.Desmos.com, Desmos}, which is an excellent tool for open-ended experimentation.
+
+=== Launch
+
+Make a scatter-plot showing the Covid infection rate for Massachusetts. What kind of model do you think would fit this best?
+
+=== Investigate
+
+[.lesson-instruction]
+- Complete the *first* section on @printable-exercise{graphing-models.adoc}, using the first screen in @starter-file{alg2-covid-desmos} (Linear Models).
```

Comment:
I think we're trying to do too much here and missing some important opportunities as a result.  

- [ ] When I see all 3 models on the same page I assume the linear model won't be very good and am less curious about trying to optimize it. I would encourage you to split this into three separate pages and 3 independent desmos files so that students are focused on one at a time. 

- [ ] Linear model page: I am not able to visually determine which of the lines I generate in desmos is a better fit. But we've already introduced them to `fit-model`, which gives concrete feedback on how strong the correlation is. And we want them to be able to define functions in pyret. Why not have them explore in Desmos and use the m and b they find to define the function? among other things, it would force students to engage with how the sliders they are dragging around connect back to the function (that they might be ignoring in Desmos).

Reply:
Done!